src/Toast/index.jsx

```diff
@@ -90,6 +94,8 @@ Toast.propTypes = {
   closeLabel: PropTypes.string,
   /** Time in milliseconds for which the `Toast` will display. */
   delay: PropTypes.number,
+  /** class names for the span wrapping the BaseToast component */
```

Comment:
"Class names for the ``BaseToast`` component" (no mention of `span`, since that can be removed)